idea/src/org/jetbrains/kotlin/idea/quickfix/ChangeVisibilityFix.kt

```diff
@@ -51,6 +54,11 @@ open class ChangeVisibilityFix(
         }
 
         element?.setVisibility(visibilityModifier)
```

Comment:
Use `SmartPsiElementPointer`. Otherwise, you can catch `PsiInvalidElementAccessException`.
```kotlin
val pointer = element?.createSmartPointer()

val originalElement = pointer?.element
if (originalElement is KtDeclaration) {
    originalElement.runOnExpectAndAllActuals { it.setVisibility(visibilityModifier) }
}

pointer?.element?.setVisibility(visibilityModifier)

val propertyAccessor = pointer?.element as? KtPropertyAccessor
```